examples/build.gradle

```diff
@@ -17,18 +17,18 @@ buildscript {
     //********************************************************************************
 
     //********************************************************************************
-    // use artifacts from Rackspace public repository
+    // use artifacts from the Gradle Plugin repository
     //--------------------------------------------------------------------------------
     repositories {
         jcenter()
         mavenCentral()
         maven {
-            url "https://maven.research.rackspacecloud.com/content/repositories/public/"
+            url "https://plugins.gradle.org/m2/"
         }
     }
 
     dependencies {
-        classpath 'org.openrepose:gradle-jaxb-plugin:2.2.1'
+        classpath 'gradle.plugin.org.openrepose:gradle-jaxb-plugin:2.2.3'
```

Comment:
The Gradle Plugin portal adds `gradle.plugin.` to your ID unless you go through a few more hoops. Their docs also state:

> As publishing to a custom group requires manual approval by the Gradle team for security reasons, we recommend not overriding the group ID unless you have an existing group ID that you wish to keep.